### PR TITLE
testing/gnome-menus: new aport

### PR DESCRIPTION
**Pull request by z3ntu**
@Cogitri Do you want to maintain this as it's a part of GNOME?

### CODE REVIEW

File: testing/gnome-menus/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,21 @@
+pkgname=gnome-menus
```
Comment (maxice8) [inline testing/gnome-menus/APKBUILD]: In the meantime add # Maintainer:

File: testing/gnome-menus/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+pkgver=3.32.0
+pkgrel=0
+pkgdesc='Library for the Desktop Menu Specification'
```
Comment (maxice8) [inline testing/gnome-menus/APKBUILD]: single quotes

File: testing/gnome-menus/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+arch="all"
+url='https://www.gnome.org/'
```
Comment (maxice8) [inline testing/gnome-menus/APKBUILD]: single quotes

File: testing/gnome-menus/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+license="GPL-2.0-or-later AND LGPL-2.0-or-later"
+depends="glib"
```
Comment (maxice8) [inline testing/gnome-menus/APKBUILD]: found automatically via providers so: ?

**Comment by Cogitri** [comment]
Sure, can do that if you don't want it

File: testing/gnome-menus/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="gobject-introspection intltool glib-dev"
+subpackages="$pkgname-dev"
+source="http://ftp.gnome.org/pub/gnome/sources/${pkgname}/${pkgver%.*}/${pkgname}-${pkgver}.tar.xz"
+
+build() {
+    ./configure --prefix='/usr' --sysconfdir='/etc' --localstatedir='/var' --disable-static
```
Comment (maxice8) [inline testing/gnome-menus/APKBUILD]: no need to be quoted

File: testing/gnome-menus/APKBUILD (hunk 3 above)
Comment (Cogitri) [inline testing/gnome-menus/APKBUILD]: I think linking to the actual homepage of gnome-menus instead of just gnome would be better

**Comment by z3ntu** [comment]
@Cogitri Should I add you to the maintainer line in this PR then?

**Comment by Cogitri** [comment]
👍

**Comment by z3ntu** [comment]
done

File: testing/gnome-menus/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+build() {
+	./configure \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--localstatedir=/var \
+		--disable-static
+	make
+}
+
+package(){
```
Comment (Cogitri) [inline testing/gnome-menus/APKBUILD]: Nitpick, but there's a space missing here

File: testing/gnome-menus/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
```
Comment (Cogitri) [inline testing/gnome-menus/APKBUILD]: You can add yourself as contributor if you want credit for your work :)

File: testing/gnome-menus/APKBUILD [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+build() {
+	./configure \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--localstatedir=/var \
+		--disable-static
+	make
+}
+
+package() {
```
Comment (Cogitri) [inline testing/gnome-menus/APKBUILD]: It doesn't have tests?

Comment (Cogitri) [reply testing/gnome-menus/APKBUILD]: If so, that should be reflected by `options="!check"`

File: testing/gnome-menus/APKBUILD (hunk 7 above)
Comment (z3ntu) [reply testing/gnome-menus/APKBUILD]: I didn't really create the APKBUILD, it's coming from the pmaports tree from postmarketOS, so it's fine :)

**Pull request closed without merging**